src/brsTypes/components/RoAssociativeArray.ts

```diff
@@ -142,7 +187,7 @@ export class RoAssociativeArray extends BrsComponent implements BrsValue, BrsIte
             returns: ValueKind.Void,
         },
         impl: (interpreter: Interpreter, key: BrsString, value: BrsType) => {
-            this.set(key, value);
+            this.set(key, value, true);
```

Comment:
can we add a comment to make it clear what this boolean refers to?
```suggestion
            this.set(key, value, /* isCaseSensitive */ true);
```